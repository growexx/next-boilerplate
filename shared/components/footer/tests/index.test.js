import { render } from '@testing-library/react'

import Footer from '../index'

describe('<Footer />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild }
    } = render(<Footer />)
    expect(firstChild).toMatchSnapshot()
  })
})
