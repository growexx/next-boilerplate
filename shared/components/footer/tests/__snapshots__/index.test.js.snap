// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render and match the snapshot 1`] = `
<footer
  class="sc-beqWaB igQRWg"
>
  <section>
    footer.copyRightMessage
    <section>
      footer.copyRightSubMessage
    </section>
  </section>
  <section />
</footer>
`;
